homeassistant/components/matter/strings.json

```
@@ -441,6 +441,9 @@
       "evse_soc": {
         "name": "State of charge"
       },
+      "pi_heating_demand": {
+        "name": "Heating Demand"
```

To follow HA conventions, only the first letter should be upper-case, except for abbreviations or proper names, so it should be like this:
```suggestion
        "name": "Heating demand"
```

Otherwise, for ZHA, we use "Pi heating demand" directly: [zha/strings.json](https://github.com/home-assistant/core/blob/b3e16bd4fa1a34f216e262912a51252a90887440/homeassistant/components/zha/strings.json#L1660)
```suggestion
        "name": "Pi heating demand"
```